zerver/models.py

```diff
@@ -1509,6 +1509,7 @@ class UserBaseSettings(models.Model):
     # This setting controls which view is rendered first when Zulip loads.
     # Values for it are URL suffix after `#`.
     default_view = models.TextField(default="recent_topics")
+    do_not_mark_messages_as_read = models.BooleanField(default=False)
```

Comment:
This name sounds like we'll never mark messages as read, which is confusing. We should give especially careful thought to the names of settings like this that will be part of the API.

Can you start a thread in "#api design" about this value? I think we'll want to discuss both what to call it, but also whether we want a slightly different format (the mobile and web apps may want the same or different configuration here, and I'm just not sure how best to manage that in terms of what this is called).

One idea would be something like `web_app_mark_read_on_scroll`, if we want to make this clearly explicitly not controlling the mobile app behavior.